File: Java/961. N-Repeated Element in Size 2N Array.java
class Solution {
    public int repeatedNTimes(int[] nums) {
        HashMap<Integer, Integer> mp = new HashMap<Integer, Integer>();
        for(int i: nums){
            mp.put(i,mp.getOrDefault((i),0)+1);
            if(mp.get(i)>1){
                return i;
            }
        }
        return 0;
    }
}

File: C++/961. N-Repeated Element in Size 2N Array.cpp
class Solution {
public:
    int repeatedNTimes(vector<int>& nums) {
        map<int, int> mp;
        for(auto i: nums){
            mp[i]++;
            if(mp[i]>1){
                return i;
            }
        }
        return 0;
    }
};
